doc/policy/version3_transactions.md

```diff
@@ -0,0 +1,100 @@
+# Transactions with nVersion 3
+
+A transaction with its `nVersion` field set to 3 ("V3 transactions") is allowed in mempool and
+transaction relay.
+
+The goal with V3 is to create a policy that retains DoS-resistance and accepts transactions that are
+incentive-compatible to mine, while making fee-bumping more robust by avoiding specific pinning attacks.
+Contract or L2 protocols in which transactions are signed by untrusted counterparties well before
+broadcast time, e.g. the Lightning Network (LN), may benefit from using this version.
+
+## V3 Rationale: Pinning Attacks
+
+Since contracting transactions are shared between multiple parties and mempool congestion is
+difficult to predict, mempool policy restrictions may accidentally allow a malicious party to pin a
+transaction. For example, imagine that counterparties Alice (honest) and Mallory (malicious) have
+conflicting transactions A and B, respectively. Mallory may be able to prevent A from being accepted
+by first submitting B and taking advantage of some limitations and heuristics used in [RBF
+policy](./mempool-replacements.md).
+
+### "Rule 3" Pinning
+
+RBF rules require the replacement transaction pay a higher absolute fee than the aggregate fees paid
+by all original transactions. This means Mallory may increase the fees required to replace B by:
+
+1. Adding transaction(s) that descend from B and pay a feerate too low to fee-bump B through CPFP.
+   For example, assuming the default descendant size limit is 101KvB and B is 1000vB paying a
+feerate of 2sat/vB, adding a 100KvB, 2sat/vB child increases the cost to replace B by 200Ksat.
+
+2. Adding a high-fee descendant of B that also spends from a large, low-feerate mempool transaction,
+   C. The child may pay a very large fee but not actually be fee-bumping B if its overall ancestor
+feerate is still lower than B's individual feerate. For example, assuming the default ancestor size
+limit is 101KvB, B is 1000vB paying 2sat/vB, and C is 99KvB paying 1sat/vB, adding a 1000vB child of
+B and C increases the cost to replace B by 101Ksat.
+
+## Version 3 Rules
+
+All existing standardness rules apply to V3. The following set of additional restrictions apply to
+V3 transactions:
+
+1. A V3 transaction can be replaced, even if it does not signal BIP125 replaceability. Other
```

Comment:
I think this implies that any 0conf acceptance softwares should be upgraded to tread V3 transactions by default as replaecable. I would say it could be valuable to notify the operators of such services.